cmd/api/src/utils/validation/rrule_validator.go

```
@@ -0,0 +1,67 @@
+	if rruleStr, ok = value.(string); !ok {
+		return append(errs, fmt.Errorf(ErrInvalidRrule, value))
+	}
+
+	//Validate that the rrule is a good rule. We're going to require a DTSTART to keep scheduling consistent.
+	//We're also going to reject UNTIL/COUNT because it will most likely break the pipeline once it's hit without being invalid
+	if rruleStr == "" {
+		return nil
+	} else if _, err := rrule.StrToRRule(rruleStr); err != nil {
+		return append(errs, fmt.Errorf(ErrInvalidRrule, err))
+	} else if strings.Contains(strings.ToUpper(rruleStr), "UNTIL") {
+		return append(errs, fmt.Errorf(fmt.Sprintf(ErrInvalidRrule, "until not supported")))
```

golangci-lint (1.64.8)

[error] SA1006: printf-style function with dynamic format string and no further arguments should use print-style function instead

(staticcheck)